src/apps/competitions/models.py

```diff
@@ -38,7 +99,36 @@ class Phase(models.Model):
     end = models.DateTimeField(null=True, blank=True)
     name = models.CharField(max_length=256)
     description = models.TextField(null=True, blank=True)
+    never_ends = models.BooleanField(default=False)
+
+    @property
+    def is_active(self):
+        """ Returns true when this phase of the competition is on-going. """
+        if self.never_ends:
+            return True
+        else:
```

Comment:
Can maybe just do something like this, since we checked never_ends? do we even need a "never ends" boolean, can't we just see if we have a `.end`?
```python
return start < timezone.now() < end
```